ts/src/bitopro.ts

```diff
@@ -136,19 +136,19 @@ export default class bitopro extends Exchange {
                         'wallet/depositHistory/{currency}',
                         'wallet/withdrawHistory/{currency}',
                     ],
-                    'post': [
-                        'orders/{pair}',
-                        'orders/batch',
-                        'wallet/withdraw/{currency}',
-                    ],
-                    'put': [
-                        'orders',
-                    ],
-                    'delete': [
-                        'orders/{pair}/{id}',
-                        'orders/all',
-                        'orders/{pair}',
-                    ],
+                    'post': {
```

Comment:
@sc0Vu what about the other endpoints? can't we add the weight to them as well?

Reply:
@carlosmiei it seems they didn't add rate limit for all endpoints. I'll check with them.

Reply:
They replied in the channel. I'll update.

Open API:
 ◦ 600 requests per minute per IP

Auth API:
 ◦ 600 requests per minute per IP
 ◦ 600 requests per minute per user account